frontend/src/components/PresentationRequestList.vue

```diff
@@ -0,0 +1,112 @@
+<!--
+ Copyright (c) 2020 - for information on the respective copyright owner
+ see the NOTICE file and/or the repository at
+ https://github.com/hyperledger-labs/organizational-agent
+
+ SPDX-License-Identifier: Apache-2.0
+-->
+
+<template>
+  <v-data-table
+    :hide-default-footer="presentationRequests.length < 10"
+    v-model="selected"
+    :show-select="selectable"
+    :headers="headers"
+    :items="presentationRequests"
+    :expanded.sync="expanded"
+    item-key="index"
+    :sort-by="['createdAt']"
+    :sort-desc="[false]"
+    single-select
+    @click:row="openPresentation"
+  >
+    <template v-slot:[`item.createdAt`]="{ item }">
+      {{ item.createdAt | formatDateLong }}
```

Comment:
The returned value from the api call looks right, but has decimal places this is always showing at 1970. should be a quick fix for a trained eye. let me know

Reply:
The filter should expect milliseconds since epoch. What's provided by the api?

Reply:
I was providing an Instant instance, and was being serialized as a float. Looks like there are view classes that convert it to Long before serialisation. So i mimiced that.